dali/test/python/test_optical_flow.py

```diff
@@ -211,33 +224,52 @@ def flow_to_color(flow_uv, clip_flow=None, convert_to_bgr=False):
 
 interactive = False
 
+def check_optflow(output_grid=1, hint_grid=1, use_temporal_hints=False):
+    batch_size = 3
+    pipe = of_pipeline(batch_size=batch_size, num_threads=3, device_id=0, output_grid=output_grid,
+                       hint_grid=hint_grid, use_temporal_hints=use_temporal_hints)
+    pipe.build()
+    if get_arch() < 8 and (output_grid != 4 or hint_grid != 4):
+        assert_raises(RuntimeError, pipe.run, "grid size : * is not supported, supported are:")
+
+    for _ in range(2):
```

Comment:
Shouldn't it be in `else:` clause?

Reply:
I thought it just ends up the test (early exit). I can add `else` if it reads better in your opinion.

Reply:
Done

Reply:
Sure, but I think it boils down to `pass` rather than `return` in that case.